ArjunaJTS/interop/glassfish/GLASSFISH-21532.diff

```diff
@@ -0,0 +1,25 @@
+Index: appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/GlobalTID.java
+===================================================================
+--- appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/GlobalTID.java	(revision 64262)
```

Comment:
I raised GLASSFISH-21532 jira.
I will raise one for WildFly once I have demonstrated recovery (JBTM-2873)